hack/enable-kibana-msearch-access.sh

```diff
@@ -0,0 +1,192 @@
+#!/bin/bash
+# this grants the given user access to _msearch on the given projects
+# the /sg/roles/0 should look like this:
+#         "hack_for_user_access_loguser": {
+#             "cluster": [],
+#             "indices": {
+#                 "multi-tenancy-1?843a78a0-5ad5-11e7-b47f-0e35fd64a858?*": {
+#                     "*": [
+#                         "indices:admin/validate/query*",
+#                         "indices:admin/get*",
+#                         "indices:admin/mappings/fields/get*",
+#                         "indices:data/read*"
+#                     ]
+#                 },
+#                 "multi-tenancy-2?84f15332-5ad5-11e7-b47f-0e35fd64a858?*": {
+#                     "*": [
+#                         "indices:admin/validate/query*",
+#                         "indices:admin/get*",
+#                         "indices:admin/mappings/fields/get*",
+#                         "indices:data/read*"
+#                     ]
+#                 },
+#                 "project?multi-tenancy-1?843a78a0-5ad5-11e7-b47f-0e35fd64a858?*": {
+#                     "*": [
+#                         "indices:admin/validate/query*",
+#                         "indices:admin/get*",
+#                         "indices:admin/mappings/fields/get*",
+#                         "indices:data/read*"
+#                     ]
+#                 },
+#                 "project?multi-tenancy-2?84f15332-5ad5-11e7-b47f-0e35fd64a858?*": {
+#                     "*": [
+#                         "indices:admin/validate/query*",
+#                         "indices:admin/get*",
+#                         "indices:admin/mappings/fields/get*",
+#                         "indices:data/read*"
+#                     ]
+#                 }
+#             }
+#         }
+
+set -euo pipefail
+
+if [ -n "${DEBUG:-}" ] ; then
+    set -x
+    curl_output() {
+        python -mjson.tool
+    }
+else
+    curl_output() {
+        cat > /dev/null 2>&1
+    }
+fi
+
+# $1 - es pod name
+# $2 - es endpoint
+# rest - any args to pass to curl
+function curl_es() {
```

Comment:
@portante This script is meant to be standalone, self contained, that can be run on site at customer without the rest of origin-aggregated-logging.  That's why I copied these into the script.